src/init/features/dataconnect/index.ts

```diff
@@ -266,24 +267,15 @@ function subConnectorYamlValues(replacementValues: { connectorId: string }): str
   return replaced;
 }
 
-async function promptForExistingServices(
-  setup: Setup,
-  info: RequiredInfo,
-  isBillingEnabled: boolean,
-): Promise<RequiredInfo> {
-  if (!setup.projectId || !isBillingEnabled) {
-    // TODO(b/368609569): Don't gate this behind billing once backend billing fix is rolled out.
```

Comment:
Got rid of the `isBillingEnabled` check here because we no longer needs it.

FDC API can be used without billing.